doc/rst/source/explain_symbols.rst_

```diff
@@ -336,8 +336,8 @@
         with the same pen as set for the line (i.e., via **-W**). To use an
         alternate pen, append **+p**\ *pen*. To skip the outline, just use
         **+p** with no argument.  To make the main front line invisible, add **+i**.  **Note**:
-        By placing **-Sf** options in the segment header you can change the front
-        types on a segment-by-segment basis.
+        By placing **-Sf** options in the segment headers that differ from the one on
+        the command line you can change the front types on a segment-by-segment basis.
```

Comment:
"font" not "front"

Reply:
**-Sf** is for drawing a front, so I think this is probably an intentional word choice

Reply:
Yes, likely.